### DISCUSSION TITLE
Partner program showcases!

### DISCUSSION THREAD

**Discussion by ctih1** [ctih1/frii.site-frontend#66] [Show and tell]
# Partner program showcases!
This is the best to share your own projects that use the frii.site API. Please read the [partner program is here](https://github.com/ctih1/frii.site-frontend/discussions/67) thread to understand everything.

## How do I submit?
Submitting is very easy! Just make a new comment that showcases your project. A GIF or  simple explanation will do. For the maximum trust, link the source code of your project. This is **NOT** mandatory though. 

## For users
Keep in mind that we cannot guarantee that these projects are a 100% safe. We advise you to run them through an antivirus software, and read other peoples comments. This system will be greatly overhauled in a few weeks, making this much safer.

**Comment by WhatDidYouExpect** [comment]
you can delete, create, list and update domain IPs the user enters their username and password and it runs it against /get-user-info to check if its valid and saves the token in a .txt file in the same directory and the other scripts send the post requests via that token

https://github.com/WhatDidYouExpect/frii-site-manager

**Comment by rock3tsprocket** [comment]
i made a frii.site cog for Discord bots
https://github.com/rock3tsprocket/cube/blob/main/cogs/friisite.py
